controllers/webhooks/services/bindings/validator.go

```diff
@@ -47,6 +47,10 @@ func NewCFServiceBindingValidator(duplicateValidator webhooks.NameValidator) *CF
 
 func (v *CFServiceBindingValidator) ValidateCreate(ctx context.Context, obj runtime.Object) (admission.Warnings, error) {
 	serviceBinding, ok := obj.(*korifiv1alpha1.CFServiceBinding)
+	if serviceBinding.Labels[korifiv1alpha1.ServiceCredentialBindingTypeLabel] == korifiv1alpha1.CFServiceBindingTypeKey {
```

Comment:
Why do we do this? If we short-cut the validation we do on create for service bindings of type key, we might get duplicate entries, as we wouldn't invoke the duplicate validator.